xAPI.md

```diff
@@ -2331,30 +2350,31 @@ of "http://adlnet.gov/expapi/attachments/signature" and a contentType of "applic
 X.509 certificate.
 * If X.509 was used to sign, the JWS header SHOULD include the "x5c" property containing
 the associated certificate chain.
+
+##### LRS Requirements
 * The LRS MUST reject requests to store Statements that contain malformed signatures, with HTTP 400.
-* The LRS SHOULD include a message in the response of a rejected statement.  
+* The LRS SHOULD include a message in the response of a rejected statement. 
 * In order to verify signatures are well formed, the LRS MUST do the following:
     * Decode the JWS signature, and load the signed serialization of the Statement from the
       JWS signature payload.
-    * Validate that the "original" Statement is logically equivalent to the received Statement.
-        * When making this equivalence check, differences which could have been caused by
-        allowed or required LRS processing of "id", "authority", "stored", "timestamp", or
-        "version" MUST be ignored.
+    * Validate that the "original" Statement is logically equivalent to the received Statement. 
+    See [Statement comparision requirements](statement-comparision-requirements).
     * If the JWS header includes an X.509 certificate, validate the signature against that
     certificate as defined in JWS.
-* Clients MUST NOT assume a signature is valid simply because an LRS has accepted it.
+    * validate that the Signature Requirements outlined above have been met. 
```

Comment:
New requirement? Not really; it was ambiguous as to whether the signature requirements related to LRS or client. We're now saying they related to both.